src/events.cpp

```diff
@@ -615,10 +616,11 @@ void pump()
 			// Here we can trigger any watchers for resize events.
 			// Video settings such as game_canvas_size() will be correct.
 			case SDL_WINDOWEVENT_RESIZED:
-				LOG_DP << "events/RESIZED "
+				LOG_DP << "events/RESIZED"
```

Comment:
Why did you remove a space here and on line 610?

Reply:
Updated.